modules/data.atmosphere/R/met.process.R

```diff
@@ -25,26 +26,120 @@ met.process <- function(site, input_met, start_date, end_date, model, host, bety
   dbparms <- list(driver=driver, user=user, dbname=dbname, password=password, host=bety.host)
   con       <- db.open(dbparms)
   
+  
+  
+  #' Determine where download and conversion will take place -  either on Brown Dog or specified directory
+  #' Would like to ultimately make "convert" even more specific:
+  #' convert = "bd_raw"
+  #'         = "dir_raw"
+  #'         = "bd_cf"
+  #          = "dir_cf"
+  #'         = "bd_model"
+  #'         = "dir_model"
   met <- input_met$source
   if(!exists("input_met$id") || input_met$id==""){
-    download=TRUE
+    if (!is.null(browndog$url) && (browndog$url != "")){
+      convert = "browndog"
+    }else{
+      convert = "dir"
+    }
   }else{
-    download=FALSE
+    convert=""
     raw.id=as.numeric(input_met$id)
   }
   
+  # Determine if met data is regional or site - this needs to be automated!
   regional <- met == "NARR" # Either regional or site run
   new.site = as.numeric(site$id)
   str_ns    <- paste0(new.site %/% 1000000000, "-", new.site %% 1000000000)
   
+  # Determine output type - this needs to be automated!
+  if("ED2" %in% unlist(strsplit(model,"[.]"))){
```

Comment:
This is only needed for BrownDog. All other met processing functions should return information about what the format type, name, and path on disk of the output is.